python-package/lightgbm/engine.py

```diff
@@ -40,7 +40,9 @@ def train(params, train_set, num_boost_round=100,
         Customized objective function.
     feval : callable or None, optional (default=None)
```

Comment:
@ClimbsRocks Please add that it could be a string.
`feval : callable, string or None, optional (default=None)`